src/table/settings.ini

```diff
@@ -3063,14 +3063,19 @@ strhelp  = STR_CONFIG_SETTING_DEFAULT_RAIL_TYPE_HELPTEXT
 strval   = STR_CONFIG_SETTING_DEFAULT_RAIL_TYPE_FIRST
 cat      = SC_BASIC
 
-[SDTC_BOOL]
-var      = gui.enable_signal_gui
+[SDTC_VAR]
+var      = gui.signal_gui_mode
+type	 = SLE_UINT8
 flags    = SLF_NOT_IN_SAVE | SLF_NO_NETWORK_SYNC
-def      = true
-str      = STR_CONFIG_SETTING_ENABLE_SIGNAL_GUI
-strhelp  = STR_CONFIG_SETTING_ENABLE_SIGNAL_GUI_HELPTEXT
+guiflags = SGF_MULTISTRING
+def      = 0
+min      = 0
+max      = 1
+str      = STR_CONFIG_SETTING_SIGNAL_GUI_MODE
+strhelp  = STR_CONFIG_SETTING_SIGNAL_GUI_MODE_HELPTEXT
+strval   = STR_CONFIG_SETTING_SIGNAL_GUI_MODE_PATH
 proc     = CloseSignalGUI
-cat      = SC_EXPERT
+cat      = SC_ADVANCED
```

Comment:
Lately we have been converting some, and most likely more will follow, where if a value represents an enum, we use their string representation in the config file. It reads a bit better to see "signal_gui_mode = path` over `1` :)
This is absolutely not trivial to do, but soon I hope to do some others which hopefully work as example. Just a heads-up more than anything else :)

Reply:
Thanks for the heads-up. If it's "absolutely not trivial" I won't try it for now. A bit over my head, probably. :)